hpe_3par_kubernetes_manager.py

```diff
@@ -310,12 +352,39 @@ def hpe_list_secret_objects_names(namespace):
         raise e
 
 
+def hpe_list_service_objects(namespace):
+    try:
+        service_list = k8s_core_v1.list_namespaced_service(namespace=namespace)
+        return service_list
+    except client.rest.ApiException as e:
+        #print("Exception :: %s" % e)
+        logging.getLogger().error("Exception :: %s" % e)
+        raise e
+
+
+def hpe_list_service_objects_names(namespace):
+    try:
+        service_names = []
+        service_list = hpe_list_service_objects(namespace=namespace)
+        for service in service_list.items:
+            service_names.append(service.metadata.name)
+
+        return service_names
+    except client.rest.ApiException as e:
+        logging.getLogger().error("Exception :: %s" % e)
+        raise e
+
+
 def hpe_list_pod_objects(namespace, **kwargs):
     try:
-        # print("kwargs :: keys :: %s,\n values :: %s" % (kwargs.keys(), kwargs.values()))
-        # print("value :: %s " % kwargs['label'])
-        # pod_list = k8s_core_v1.list_namespaced_pod(namespace=namespace, label_selector=kwargs['label'])
-        pod_list = k8s_core_v1.list_namespaced_pod(namespace=namespace)

```

Comment:
Please provide a valid reason for commenting on these lines of code.

Reply:
The lines that were removed were already commented, as the pod_list was only calling the basic "list_namespaced_pod" without any parameters other than namespace. The change now is if a label or field is passed, we can pass that to the method to get the pod list for those specific pods. If none are passed, proceed as it was before.